src/main/java/codeu/controller/ChatServlet.java

```diff
@@ -117,6 +118,7 @@ public void doPost(HttpServletRequest request, HttpServletResponse response)
       throws IOException, ServletException {
 
     String username = (String) request.getSession().getAttribute("user");
+    request.setAttribute("image", "like.png");
```

Comment:
Doesn't look like "image is used in the jsp, could we remove this & the png for now?